versioned_docs/version-v2.18.x/user-guide/configure-apiml-zosmf-workflow.md

```diff
@@ -0,0 +1,118 @@
+# Configuring API ML with z/OSMF Workflows
+
+After you install Zowe, you can register and execute the z/OSMF workflows in the web interface to perform a range of
+API ML/Zowe configuration tasks. The **Stand-alone Zowe API ML Configuration** workflow simplifies configuration of Zowe API Mediation Layer and does not require the level of
+expertise that is needed to perform manual API ML/Zowe configuration. The **Stand-alone Zowe API ML Configuration** workflow also runs the `zwe install` and `zwe init` command to initialize Zowe z/OS runtime.
+
+:::info Required role: system programmer
+:::
+
+Ensure that you meet the following requirements before you start your Zowe configuration:
+
+- Install and configure z/OSMF
+- Install Zowe with an SMP/E build, PSWI, or a convenience build
+
+The following components are automatically enabled after performing the **Stand-alone Zowe API ML Configuration** workflow:
+
+- Zowe configuration manager
+- Gateway
+- Gateway internal mapper
+- API Gateway
+- API Catalog
+- Discovery service
+- Caching service
+
+The following components are automatically disabled:
+
+- Metrics service
+- Cloud gateway
+- Application server
+- Jobs API
+- Files API
+- ZSS component
+- JES Explorer
+- MVS Explorer
+- USS Explorer
+  
+:::note
+These components can be enabled by performing manual changes in the yaml file, but making such changes to the yaml file is not recommended for simplified and optimized configuration of API ML.
+:::
+
+You can execute the API ML/Zowe configuration workflow either from a PSWI during deployment or later from a created software
+instance in z/OSMF. Alternatively, you can execute the configuration workflow z/OSMF during the workflow registration process.
```

Comment:
Is "configuration workflow z/OSMF" the correct name of the workflow?

Reply:
there is missing filler, probably better formulation is: you can execute the configuration workflow **in** z/OSMF during the workflow registration process